apps/epic-web/src/styles/commerce.css

```diff
@@ -0,0 +1,646 @@
+#__next {
+  [data-pricing-container] {
+    @apply mx-auto flex w-full max-w-screen-lg flex-wrap justify-center gap-5;
+    #main-pricing {
+      @apply w-full max-w-sm;
+      [data-pricing-product] {
+        [data-pricing-product-image] {
+          @apply relative mx-auto -mb-32 h-56 w-56;
+        }
+        article {
+          @apply rounded-lg border border-gray-100 bg-white pt-36 shadow-2xl shadow-gray-500/10;
+        }
+        [data-pricing-product-header] {
+          [data-byline] {
+            @apply hidden;
+          }
+          [data-name-badge] {
+            @apply pb-2 text-center  text-sm font-extrabold uppercase text-brand;
```

Comment:
the `font-heading` class was throwing an error so I removed it. COuldn't figure it out!

Reply:
yeah the class does not exist in epic web